pycaret/internal/plots/time_series.py

```diff
@@ -1025,3 +1025,197 @@ def plot_time_series_decomposition(
 
     return fig, return_data_dict
 
+
+def plot_multiple_predictions(
+        data: pd.Series,
+        predictions: Union[pd.Series, List[pd.Series]],
+        model_name: List = None,
+        data_kwargs: Optional[Dict] = None,
+        fig_kwargs: Optional[Dict] = None,
+):
+    """Plots the original data and the predictions provided with confidence"""
+    fig, return_data_dict = None, None
+
+    if data_kwargs is None:
+        data_kwargs = {}
+    if fig_kwargs is None:
+        fig_kwargs = {}
+
+    title = "Actual vs. 'Out-of-Sample' Forecast"
+    time_series_name = data.name
+    if time_series_name is not None:
+        title = f"{title} | {time_series_name}"
+
+    fig_data = []
+    for index_iterator in range(len(predictions)):
+        x = (
+            predictions[index_iterator].index.to_timestamp()
+            if isinstance(predictions[index_iterator].index, pd.PeriodIndex)
+            else predictions[index_iterator].index
+        )
+        mean = go.Scatter(
+            name=f"Forecast | {model_name[index_iterator]}",
+            x=x,
+            y=predictions[index_iterator],
+            mode="lines+markers",
+            showlegend=True,
+        )
+
+        fig_data.extend([mean])
+
+    x = (
+        data.index.to_timestamp()
+        if isinstance(data.index, pd.PeriodIndex)
+        else data.index
+    )
+
+    original = go.Scatter(
+        name="Original",
+        x=x,
+        y=data,
+        marker=dict(size=5, color="#3f3f3f"),
+        mode="lines+markers",
+        showlegend=True,
+    )
+    fig_data.append(original)
+
+    layout = go.Layout(
+        yaxis=dict(title="Values"), xaxis=dict(title="Time"), title=title,
+    )
+
+    fig = go.Figure(data=fig_data, layout=layout)
+
+    fig_template = fig_kwargs.get("fig_template", "ggplot2")
+    fig.update_layout(template=fig_template)
+    fig.update_layout(showlegend=True)
+
+    fig_size = fig_kwargs.get("fig_size", None)
+    if fig_size is not None:
+        fig.update_layout(
+            autosize=False, width=fig_size[0], height=fig_size[1],
+        )
+
+    return_data_dict = {
+        "data": data,
+        "predictions": predictions,
+        "model_name":model_name
+    }
+
+    return fig, return_data_dict
+
+
+def plot_multiple_predictions_with_confidence(
```

Comment:
Can we merge this with `plot_multiple_predictions` since there seems to be lot of overlap.
You can assume that is `upper_interval` or `lower_interval` = None, then no confidence interval needs to be plotted